### PR TITLE
FIX: Add missing translations

### PR DESCRIPTION
**Pull request by 1kuemmel1**
- FIX: Add missing translations (@1kuemmel1).

**When merged this pull request will:**
- add some missing translations
- correcting some spacing

**Final test:**
- [ ] local
- [ ] server

### CODE REVIEW

**Comment by Vdauphin** [comment]
Hello,
This is a FIX so use the master_stable
See you soon!

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 1]
```diff
@@ -302,15 +302,19 @@
...
                 <Original>Very easy</Original>
+                <German>Sehr leicht</German>
             </Key>
             <Key ID="STR_BTC_HAM_PARAM_IED_SPOT_EASY">
                 <Original>Easy</Original>
+                <German>Leicht</German>
             </Key>
             <Key ID="STR_BTC_HAM_PARAM_IED_DRONE">
                 <Original>Drone bomber:</Original>
+                <German>Feindl. Dronen mit Bomben:</German>
```
Comment (bux) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: Drohnen

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 2]
```diff
@@ -604,56 +608,56 @@
...
+                <Original>Civil need help</Original>
+                <German>Zivilist braucht Hilfe</German>
+            </Key>
+            <Key ID="STR_BTC_HAM_SIDE_CIVTREAT_TITLE">
+                <Original>Medical emergency call in %1</Original>
+                <German>Medizinischer Notruf aus %1</German>
+            </Key>
+            <Key ID="STR_BTC_HAM_SIDE_CIVTREAT_DESC">
+                <Original>A civilian is calling for a medic in %1, treat and wait for patient stabilization </Original>
+                <German>Ein Zivilist aus %1 hat einen Notruf abgesetzt. Behandeln Sie ihn umd warten Sie darauf das sein Zustand sich stabilisiert </German>
```
Comment (bux) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: ... darauf, dass sich sein ...

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml (hunk 1 above)
Comment (1kuemmel1) [reply =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: 😕... thanks for the hint.

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml (hunk 2 above)
Comment (1kuemmel1) [reply =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: *", dass"* mein Feind aus Kindertagen. 😆 

Erneut Danke 👍

**Comment by 1kuemmel1** [comment]
Open a new PR; see #499

**Pull request closed without merging**